### PR TITLE
feat: add landing and documentation page

### PR DESCRIPTION
**Pull request by Kavita539**
Added landing page and documentation page.
Made it responsive.

Landing page :
image

Documenation->Introduction/Installation page : 
image

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
looks good to me

**Comment by Kavita539** [comment]
> looks good to me

Thanks!

File: index.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    color: white;
+    text-align: center;
+}
+
+.btn-secondary{
+    background-color: var(--secondary-color);
+    color : var(--primary-color);
+    border: 1px solid var(--primary-color);
+    text-align: center;
+}
```
Comment (devananth) [inline index.css]: Can move this `text-align:center` to `.btn` class

File: stylesheets/base/reset.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-size: 1rem;
+    font-family: 'Roboto', sans-serif;
+}
+
+body {
+    /* color: #f5f5f4; */
+    /* color: #676e8b; */
```
Comment (devananth) [inline stylesheets/base/reset.css]: you can remove the commented lines

File: stylesheets/layout/navbar.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
+@import url(../stylesheets/base/root-variables.css);
+@import url(../stylesheets/base/reset.css);
+
+.navbar{
+   top: 0;
+   z-index: 3;  
+   display: flex;
+   justify-content: space-between;
+   align-items: center;
+   /* flex-direction: row; */
```
Comment (devananth) [inline stylesheets/layout/navbar.css]: Can remove this too

File: stylesheets/layout/navbar.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    justify-content: space-evenly;
+    gap: .5rem;
+    align-items: center;
+ }
+
+/* .left-nav{ */
+   /* font-size: 1.7rem; */
+   /* font-weight: bold; */
+   /* color: var(--logo-color); */
+/* } */
```
Comment (devananth) [inline stylesheets/layout/navbar.css]: This too can be removed , if not necessary

**Review by devananth — COMMENTED** [review]
Good work !

File: index.css (hunk 1 above)
Comment (Kavita539) [reply index.css]: ok thanks, will do!

File: stylesheets/layout/navbar.css (hunk 4 above)
Comment (Kavita539) [reply stylesheets/layout/navbar.css]: yeah will remove all the commented and unnecessary lines!

**Review by msdeshmukh009 — APPROVED** [review]

**Review by devananth — COMMENTED** [review]
You can merge it

**Pull request merged**